pr2eus_tutorials/launch/pr2_tabletop.launch

```diff
@@ -5,11 +5,25 @@
   <arg name="manager" default="pr2_tabletop_object_detector_nodelet_manager"/>
   <arg name="machine" default="c2" />
   <arg name="load_machine" default="false" />
+  <arg name="remote" default="false" />
+
+  <arg name="RGB_CAMERA_INFO" value="/kinect_head/rgb/camera_info" />
+  <arg name="RGB_IMAGE" value="/kinect_head/rgb/image_rect_color" />
+  <arg name="RGB_IMAGE_REMOTE" value="/kinect_head_remote/rgb/image_rect_color" />
+  <arg name="DEPTH_CAMERA_INFO" value="/kinect_head/depth_registered/camera_info" />
+  <arg name="DEPTH_IMAGE" value="/kinect_head/depth_registered/image_rect" />
+  <arg name="DEPTH_IMAGE_REMOTE" value="/kinect_head_remote/depth_registered/image_rect" />
+  <arg name="PROSILICA_CAMERA_INFO" value="/prosilica/camera_info" />
+  <arg name="PROSILICA_IMAGE" value="/prosilica/image_raw" /> 
+  <arg name="PROSILICA_IMAGE_REMOTE" value="/prosilica_remote/image_raw" /> 
+  <arg name="POINT_CLOUD" value="/kinect_head/depth_registered/throttled/points" />
+  <arg name="POINT_CLOUD_REMOTE" value="/kinect_head_remote/depth_registered/points" />
```

Comment:
If you use `kinect_head` and `kinect_head_remote` many times, you can use <group ns=""> tag
(If you do not like the group tag, I think it's ok not to use the group tag)

example:
https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_pcl_ros/launch/openni2_remote.launch#L80-L85

Reply:
I agree with you.

Reply:
Fix with [818368320ff6a5683d1062d5d67174d3bd74ab04](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/818368320ff6a5683d1062d5d67174d3bd74ab04)